new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Kubernetes.java

```diff
@@ -1327,5 +1536,255 @@ public static boolean deleteClusterRoleBinding(String name) {
+          TIMEOUT_SECONDS, // Integer | Timeout for the list/watch call.
+          Boolean.FALSE // Boolean | Watch for changes to the described resources
+      );
+    } catch (ApiException apex) {
+      logger.warning(apex.getResponseBody());
+      throw apex;
+    }
+    return roleBindings;
+  }
+
+  /**
+   * Delete role in the given namespace.
+   *
+   * @param namespace name of the namespace
+   * @param name name of the role
+   * @return return true if deletion is successful
+   * @throws ApiException when delete fails
+   */
+  public static boolean deleteNamespacedRoleBinding(String namespace, String name)
+      throws ApiException {
+    try {
+      rbacAuthApi.deleteNamespacedRoleBinding(
+          name, // String | name of the job.
+          namespace, // String | name of the namespace.
+          PRETTY, // String | pretty print output.
+          null, // String | When present, indicates that modifications should not be persisted.
+          GRACE_PERIOD, // Integer | The duration in seconds before the object should be deleted.
+          null, // Boolean | Deprecated: use the PropagationPolicy.
+          FOREGROUND, // String | Whether and how garbage collection will be performed.
+          null // V1DeleteOptions.
+      );
+    } catch (ApiException apex) {
+      logger.warning(apex.getResponseBody());
+      throw apex;
+    }
+    return true;
+  }
+
+  /**
+   * List role bindings in a given name space.
+   *
+   * @param namespace name of the namespace
+   * @return V1RoleBindingList list of {@link V1RoleBinding} objects
+   * @throws ApiException when listing fails
+   */
+  public static V1RoleBindingList listNamespacedRoleBinding(String namespace)
+      throws ApiException {
+    V1RoleBindingList roleBindings;
+    try {
+      roleBindings = rbacAuthApi.listNamespacedRoleBinding(
+          namespace, // String | namespace.
+          PRETTY, // String | If 'true', then the output is pretty printed.
+          ALLOW_WATCH_BOOKMARKS, // Boolean | allowWatchBookmarks requests watch events with type "BOOKMARK".
+          null, // String | The continue option should be set when retrieving more results from the server.
+          null, // String | A selector to restrict the list of returned objects by their fields.
+          null, // String | A selector to restrict the list of returned objects by their labels.
+          null, // Integer | limit is a maximum number of responses to return for a list call.
+          RESOURCE_VERSION, // String | Shows changes that occur after that particular version of a resource.
+          TIMEOUT_SECONDS, // Integer | Timeout for the list call.
+          Boolean.FALSE // Boolean | Watch for changes to the described resources.
+      );
+    } catch (ApiException apex) {
+      logger.warning(apex.getResponseBody());
+      throw apex;
+    }
+    return roleBindings;
+  }
+
+
+  /**
+   * Delete role in the Kubernetes cluster.
```

Comment:
a role
role or clusterrole? javadoc and code don't match

Reply:
fixed